### ISSUE TITLE
Enhancement: IP Filtering to limit Jukebox requests

### ISSUE THREAD

**Issue by spankurmonkey** [whitesoup12/remote-falcon-tracker#71]
Add a feature that will allow for IP filtering to limit the number of jukebox requests a given IP address can request in a day.

**Comment by ChonLee** [comment]
Also, IP logs. I've got some joker requesting songs when they aren't around.  I was sitting out watching my show while nobody was around and songs were being requested, even at my .2 distance.

**Comment by jdesign21** [comment]
Same thing happen to me on Saturday since that's the busy day. I was outside no cars when I notice the song change out of order. Login on to the website sure enough someone queue 4 songs since that's my limit. They did it  a few times that night and i have my distance set at .1.

**Comment by md24wildcat** [comment]
I agree this feature would be great, especially if a customized error message would pop up when said IP has requested over x number of songs within y number of hours. My neighbors kid (I have been informed) likes to go to the webpage and request all songs.

Love the program so far!

**Comment by Mark79864** [comment]
> if a customized error message would pop up when said IP has requested over x number of songs within y number of hours.

I wouldn't actually bother with a message to display an IP request maxout. To me, that would entice them to switch on mobile data or turn a VPN on.

A generic message for "Sorry, something went wrong" seems more appropriate so they don't get the direct message that they have voted too much.
